tox.ini

```diff
@@ -19,7 +19,7 @@ commands =
 deps =
     flake8
 commands =
-    flake8 kolibri
+    flake8 --exclude=migrations kolibri
```

Comment:
If you put it in setup.cfg, it will also work as the default setting when running `flake8` from command line :)